solr/core/src/java/org/apache/solr/cloud/api/collections/SplitShardCmd.java

```diff
@@ -80,6 +80,15 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+/**
+ * Credits to Megan Carey for documentation
+ *
+ * <p>Index split request processed by Overseer. Requests from here go to the host of the parent
```

Comment:
In Solr 9, the Overseer doesn't necessarily process this; right?

Reply:
yes, processed by `DistributedCollectionConfigSetCommandRunner`

Reply:
Then let's not say it's processed by the Overseer when it might not be.